ci_framework/playbooks/ceph.yml

```diff
@@ -112,6 +112,8 @@
     cifmw_ceph_client_vars: /tmp/ceph_client.yml
     cifmw_cephadm_bootstrap_host: "{{ ansible_hostname }}"
     cifmw_cephadm_default_container: true
+    storage_network_range: 192.168.122.0/24
```

Comment:
Do we need to set this var? I thought we'd get this value from so that an IP range is only hard coded once and in the previous play. 
```
cifmw_cephadm_rgw_network: "{{ lookup('ansible.builtin.ini', 'public_network', section='global',  file=cifmw_cephadm_bootstrap_conf) }}"
```

Reply:
The previous comment applies to these variables (e.g. `storage_network_range` and `all_addresses`) that are not available in this playbook.
Maybe I don't need `storage_network_range` because I can get this info from `cifmw_cephadm_bootstrap_conf`. I'll try that and eventually update the patch accordingly.

Reply:
Done, as mentioned, now we get the `storage_network_range` from the Ceph `public_network` section.